ops/instrument_pam.go

```diff
@@ -102,10 +103,20 @@ func PAMTracingHandler(tracerName string, attributes ...attribute.KeyValue) pipe
 	}
 }
 
-// getRequestName, return "GetSession" from "GetBalanceRequest"
+// getRequestName, return "vplugin.PluginPAM/GetSession" from "GetBalanceRequest"
```

Comment:
Is it suppose to be `"...GetBalance" from "GetBalanceRequest"`?

Reply:
Yep.. Totally missed that they didn't match 😆